java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3918,6 +3914,106 @@ public <ArgType> T xpending(
         return getThis();
     }
 
+    /**
+     * Returns information about the stream stored at key <code>key</code>.<br>
+     * To get more detailed information use {@link #xinfoStreamFull(String)} or {@link
+     * #xinfoStreamFull(String, int)}.
+     *
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @return Command Response - A <code>Map</code> of stream information for the given <code>key
+     *     </code>.
+     */
+    public T xinfoStream(@NonNull String key) {
```

Comment:
use `ArgType` instead of string/glidestring in transaction and add implNote from #1820